crates/iota-move-build/src/lib.rs

```diff
@@ -275,42 +338,22 @@ pub fn build_from_resolution_graph(
         }
     }
 
-    let result = if print_diags_to_stderr {
-        BuildConfig::compile_package(&resolution_graph, &mut std::io::stderr())
-    } else {
-        BuildConfig::compile_package(&resolution_graph, &mut std::io::sink())
-    };
-    // write build failure diagnostics to stderr, convert `error` to `String` using
-    // `Debug` format to include anyhow's error context chain.
-    let (package, fn_info) = match result {
-        Err(error) => {
-            return Err(IotaError::ModuleBuildFailure {
-                error: format!("{error:?}"),
-            });
-        }
-        Ok((package, fn_info)) => (package, fn_info),
-    };
+    Ok(bytecode_deps)
+}
 
-    if run_bytecode_verifier {
-        let compiled_modules = package.root_modules_map();
-        for m in compiled_modules.iter_modules() {
-            move_bytecode_verifier::verify_module_unmetered(m).map_err(|err| {
-                IotaError::ModuleVerificationFailure {
-                    error: err.to_string(),
-                }
-            })?;
-            iota_bytecode_verifier::iota_verify_module_unmetered(m, &fn_info)?;
-        }
-        // TODO(https://github.com/iotaledger/iota/issues/69): Run Move linker
+/// Check that the compiled modules in `package` are valid
+fn verify_bytecode(package: &MoveCompiledPackage, fn_info: &FnInfoMap) -> IotaResult<()> {
+    let compiled_modules = package.root_modules_map();
+    for m in compiled_modules.iter_modules() {
+        move_bytecode_verifier::verify_module_unmetered(m).map_err(|err| {
+            IotaError::ModuleVerificationFailure {
+                error: err.to_string(),
+            }
+        })?;
+        iota_bytecode_verifier::iota_verify_module_unmetered(m, fn_info)?;
     }
-
-    Ok(CompiledPackage {
-        package,
-        published_at,
-        dependency_ids,
-        bytecode_deps,
-        dependency_graph: resolution_graph.graph,
-    })
+    // TODO(https://github.com/iotaledger/iota/issues/69): Run Move linker
```

Comment:
This was wrongly changed before
```suggestion
    // TODO(https://github.com/iotaledger/sui/issues/69): Run Move linker
```

Reply:
Fixed in: 37fcd716cd429717bcdef2e4de913e8ea744d9a4